src/Microsoft.AspNet.Session/SessionDefaults.cs

```diff
@@ -5,9 +5,19 @@
 
 namespace Microsoft.AspNet.Session
 {
+    /// <summary>
+    /// Represents defualts for the Session.
+    /// </summary>
     public static class SessionDefaults
     {
+        /// <summary>
+        /// Represent the default cookie name.
```

Comment:
I planned to put them before, but I don't know why I didn't wrote them as a part of the docs :(